src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -3,11 +3,14 @@
 	import Icon from '@svizzle/ui/src/icons/Icon.svelte';
 	import ChevronLeft from '@svizzle/ui/src/icons/feather/ChevronLeft.svelte';
 	import ChevronRight from '@svizzle/ui/src/icons/feather/ChevronRight.svelte';
+	import Minus from '@svizzle/ui/src/icons/feather/Minus.svelte';
+	import Plus from '@svizzle/ui/src/icons/feather/Plus.svelte';
 
 	import ColorClear from './glyphs/ColorClear.svelte'
 	import FormatClear from './glyphs/FormatClear.svelte'
 
 	import * as _ from 'lamb';
+	import {isNotNil, isNumber} from '@svizzle/utils';
```

Comment:
minor, these 2 imports need to be near the npm imports above, sorted by module name, before local imports